village/src/components/SmurfForm.js

```diff
@@ -16,15 +16,25 @@ class SmurfForm extends Component {
   }
 
   addSmurf(event) {
-    event.preventDefault();
     // add code to create the smurf using the api
-    
+    const {name, age, height} = this.state;
+
+    axios.post('http://localhost:3333/smurfs' , {
+      name: name,
+      age: age,
+      height: height
+    })
+    .then((response)=> {
     this.setState({
       name: '',
       age: '',
       height: ''
     });
-  }
+  })
+     .catch(error => {
```

Comment:
You cannot use `this.setState()` without having an initial state set in the component where it is called.